llvm/utils/TableGen/X86ManualFoldTables.def

```diff
@@ -227,6 +227,13 @@ NOFOLD(MMX_MOVQ64rr_REV)
 NOFOLD(INSERTPSrr)
 NOFOLD(VINSERTPSZrr)
 NOFOLD(VINSERTPSrr)
+// Memory faults are suppressed for CFCMOV with memory operand.
+NOFOLD(CFCMOV16rr)
+NOFOLD(CFCMOV32rr)
+NOFOLD(CFCMOV64rr)
+NOFOLD(CFCMOV16rr_ND)
+NOFOLD(CFCMOV32rr_ND)
+NOFOLD(CFCMOV64rr_ND)
```

Comment:
I think we don't need this at all.  `mayFoldFromForm` in X86FoldTablesEmitter.cpp should already returns false for `MRMDestRegCC`